backend/src/plugins/ModActions/functions/actualKickMemberCmd.ts

```diff
@@ -77,6 +77,13 @@ export async function actualKickMemberCmd(
     },
   });
 
+  if (pluginData.config.getForMember(msg.member).delete_invites_on_kick) {
+    const userInvites = (await pluginData.guild.getInvites()).filter(inv => inv.inviter?.id === user.id);
+    userInvites.forEach(invite => {
+      invite.delete("User kicked from server");
```

Comment:
This should either have a perm check above it, a `noop` catch, or a BOT_ALERT if the deletion fails

Reply:
BOT_ALERT added with 6f3aad5